py4DSTEM/process/diffraction/crystal.py

```diff
@@ -139,6 +149,69 @@ def calculate_lattice(self):
             self.pymatgen_available = True            
         else:
             self.pymatgen_available = False
+
+    def get_strained_crystal(
+        self,
+        exx = 0.0,
+        eyy = 0.0,
+        ezz = 0.0,
+        exy = 0.0,
+        exz = 0.0,
+        eyz = 0.0,
+        deformation_matrix = None,
+        return_deformation_matrix = False,
+        ):
+        """
+        This method returns new Crystal class with strain applied. The directions of (x,y,z)
+        are with respect to the default Crystal orientation, which can be checked with
+        print(Crystal.lat_real)
```

Comment:
Should we add which notation we expect users to pass or the meaning of the floats i.e. 0.01 is a 1% strain